zookeeper/Dockerfile

```diff
@@ -123,6 +123,9 @@ EOF
 
 ENV ZOOKEEPER_HOME=/stackable/zookeeper
 ENV PATH="${PATH}":/stackable/zookeeper/bin
+# This is used by zkEnv.sh and for the shell scripts in bin/
+# If unset it tries to find the conf directory automatically and that fails
+ENV ZOOCFGDIR=/stackable/config
```

Comment:
Hmm, we have a `rwconfig` as well for the real config https://github.com/stackabletech/zookeeper-operator/blob/e370101dfe986e862c0354f6a50471b857d0c078/rust/operator-binary/src/zk_controller.rs#L927.
Its created by the operator though...maybe this should be set in the operator?